table/block.h

```diff
@@ -395,8 +395,12 @@ class DataBlockIter final : public BlockIter<Slice> {
 
   virtual void Next() override;
 
+  void SlowNext();
+
   virtual void SeekToFirst() override;
 
+  void SlowSeekToFirst();
```

Comment:
How about call it `SeekToFirstCheck()` to be more accurate and cooler. Similarly, we can do `NextCheck()`. Or `SlowSeekToFirst()`. Add a comment what this is for. Explaining it is to make sure the code won't segfault even if the block is corrupted.

Reply:
Done.

Reply:
Renamed to `SeekToFirstOrReport()` and `NextOrReport`.